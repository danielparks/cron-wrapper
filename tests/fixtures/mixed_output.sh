#!/bin/bash

echo -n 111
sleep 0.1
echo -n aaa >&2
sleep 0.1
echo 333
sleep 0.1
echo bbb >&2

# Exit with the first value passed or 0.
exit ${1:-0}
