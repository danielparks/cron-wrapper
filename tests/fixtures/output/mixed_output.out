Command: ["tests/fixtures/mixed_output.sh"]
Start: 2023-07-26T13:32:22.710025000-07:00

0.003 out 111\
0.111 err aaa\
0.219 out 333
0.327 err bbb
0.327 exit 0
